entrypoint.sh

```diff
@@ -20,6 +20,15 @@ fi
 # Initialize an array to store global options
 declare -a global_options
```

Comment:
move complete global options down to current place

Reply:
done